sdk/storage/Azure.Storage.Blobs/src/RollingBufferStream.cs

```diff
@@ -0,0 +1,213 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+
+namespace Azure.Storage.Blobs
```

Comment:
Does this have to be in `Blobs` or can it move into the `Blobs.Cryptography` assembly?

Reply:
It doesn't have to be, but I feel like it's general purpose enough that it deserves to be accessible if/when other packages need it. Honestly, looking at it again, I feel it should be in `Common`.

Reply:
I'd rather move it when other packages actually need it.  This should move into the Crypto packages.